conf/flight_plans/flight_plan.dtd

```diff
@@ -65,7 +65,9 @@ ground_alt CDATA #REQUIRED
 security_height CDATA #REQUIRED
 alt CDATA #REQUIRED
 qfu CDATA #IMPLIED
-home_mode_height CDATA #IMPLIED>
+home_mode_height CDATA #IMPLIED
+max_altitude CDATA #IMPLIED
+airspace CDATA #IMPLIED>
```

Comment:
This attribute doesn't seem to be used at the moment. Also I would suggest `home_mode_max_alt` to make it clear that going higher will make the plane enter HOME mode, and `home_mode_sector` instead of airspace (if I get it correctly, this was suppose to specify the actual sector used for home protection).